packages/metal-quartz-link/src/MetalQuartzLink.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+import Component from 'metal-component';
+import MetalQuartzIcon from 'metal-quartz-icon';
+import Soy from 'metal-soy';
+import { core, object } from 'metal';
+import { validators } from 'metal-state';
+
+import templates from './MetalQuartzLink.soy.js';
+
+/**
+ * Implementation of the Metal Quartz Link.
+ */
+class MetalQuartzLink extends Component {
+	/**
+	 * @inheritDoc
+	 */
+	created() {
+		MetalQuartzLink.instances.push(this);
```

Comment:
I'm curious what the use case of this would be? If it's something we need, we'll probably need to also remove it from the array in the `disposed` lifecycle method.